configs/nextflow.ci.config

```diff
@@ -8,6 +8,7 @@ params {
 	mlstBlastDb = "${root}/assets/mlst_db/blast"
 	pubMlstDb = "${root}/assets/mlst_db/pubmlst"
 	krakenDb = "/tmp/jsn" //edit if useKraken = true
+	hostileIdx = "${root}/assets/hostile_db/human-t2t-hla"
```

Comment:
This is identical to the base config. Is it needed to duplicate this code?

Reply:
See comment below